### ISSUE TITLE
Redshift overrides xrandr brightness command

### ISSUE THREAD

**Issue by planemad** [jonls/redshift#175]
My dual external monitor does not have display adjustment buttons, but needs to be configured via a windows app. On my ubuntu machine, I use [xrandr](https://wiki.archlinux.org/index.php/xrandr) to do the job and dim the brightness using the command:
```
xrandr --output VGA1 --brightness 0.4
```
As soon as its run, the monitors dims and whitens to 6500K Kelvin, and a second later redshift kicks in restores the color temperature and brightens the screen again.

Any idea how I could dim the monitor?

**Comment by lkraav** [comment]
Yeah, same thing happens for https://github.com/zoltanp/xrandr-invert-colors and it's perfectly logical sequence of events.

Haven't had much time to figure out a solution.

**Comment by jonls** [comment]
@planemad Redshift has a brightness option, have you tried that?

**Comment by jonls** [comment]
Also, see #143 for some discussion on other software competing for the gamma ramps. In short, we cannot lock the gamma ramps so there is nothing we can do to disable the gamma adjustments made by other programs or reapply them on top of the redness effect.

There is a new option in the next release of Redshift which allows you to apply the Redshift effect on top of the existing gamma ramps. I'm not sure if that would help in your case though.

**Comment by planemad** [comment]
@jonls thanks for the brightness tip, will test it tonight.

**Comment by entrity** [comment]
@jonls : has that [new option](https://github.com/jonls/redshift/issues/175#issuecomment-73639195) allowing Redshift to be applied on top of gamma ramps gone into release yet?

I find the inability to use `xrandr` rather a handicap for my external monitor, whose brightness controls are terrible.

Alternatively, could Redshift be configured differently on each monitor? (I.e. allowing me to set a different brightness for the external monitor than for the internal one?)

**Comment by mirh** [comment]
> On my ubuntu machine, I use xrandr to do the job and dim the brightness using the command

Ehrm.. I hope you are aware that command just lowers the intensity of the pixels/colours "in the framebuffer", and not in the *physical* screen (which effectively keeps the same level of backlighting)

Anyway, if you want to rely on xrandr, there are ways to use it completely in place of other tools
https://askubuntu.com/questions/1003101/how-to-use-xrandr-gamma-for-gnome-night-light-like-usage

**Comment by lake-effect** [comment]
What's the current status of this issue? 

During the day I need to set the brightness with something like `xrandr --output eDP-1 --brightness 1.1` for readability, but then I have to snooze Redshift repeatedly to maintain the setting. I know I can "make this work" by only using Redshift at night, but then I lose all the utility of having it automatically manage the schedule. So I run into the same problems still.

**Comment by pdolinic** [comment]
Same here for what @lake-effect mentioned, any way to get around this?

Edit: Found a solution, you can pipe it like this:
```
xrandr --output eDP-1 --brightness 0.5 | redshift -O 4500
```
